docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,14 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift integrated Docker registry
+
+If you are using Fedora/CentOS/RHEL host and you are running {project} with the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] 
+then even after following the registry xref:../openshift/openshift-docker-registry.adoc#[documents], 
```

Comment:
"even after following the steps defined in xref:../openshift/openshift-docker-registry.adoc#[Accessing the OpenShift Docker Registry]"

"documents" does not make much sense  in this conext.